avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -71,64 +71,64 @@ Update the given readme file's 'Resource Types' section based on the given templ
 #>
 function Set-ResourceTypesSection {
 
-    [CmdletBinding(SupportsShouldProcess)]
```

Comment:
Hey @ReneHezser, any idea why your local pwsh instance changed the indent from 4 to 2 even though 4 is set in the vscode settings.json?

Reply:
Good questions. The settings jumps back to 2 after I set it to 4 manually. Strange. But I could commit with 4.

Reply:
That would be appreciated. Maybe it's a workspace setting vs. a general setting situation (or multiple extensions conflicting).
Would be good to sort this out, as a future PR from somebody else would otherwise again show all the whitespace changes...